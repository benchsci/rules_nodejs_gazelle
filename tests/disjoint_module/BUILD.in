# gazelle:js_root
# gazelle:js_collect_barrels
# gazelle:js_quiet
