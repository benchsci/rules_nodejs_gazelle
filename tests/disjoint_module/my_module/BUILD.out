load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

ts_project(
    name = "my_module",
    srcs = [
        "index.ts",
        "module_file.ts",
    ],
    tags = ["js_barrel"],
)

ts_project(
    name = "some_other_file",
    srcs = ["some_other_file.ts"],
)
